project_task_date_shift/models/project.py

```diff
@@ -0,0 +1,84 @@
+from dateutil.relativedelta import relativedelta
+from odoo import models, fields, api
+
+
+class Project(models.Model):
+    _inherit = 'project.project'
+
+    old_start_date = fields.Date(string='Old Start Date',
+                                 help="Used by the Shift Dates function. \

```

Comment:
I think this character is wrong \^M

Reply:
I don't know of another way to break the long help description up to be pep8 compliant and '\' seemed to work for all the other modules I've built. Any other suggestion?

Reply:
I think you don't need any extra character like in this example:
`        help="For records searched by name (taxes, fiscal "
             "positions), the template name will be matched against the "
             "record name on this language.",`

Reply:
Thanks, i've updated the code and it seems to be working correctly.